spec/spans.yml

```
@@ -0,0 +1,174 @@
+groups:
+  - id: span.graphql.server.operation
+    type: span
+    stability: development
+    span_kind: server
+    brief: >
+      This span represents an incoming GraphQL operation on a server implementation.
```

Relate to my comment about the `id`: if this span represents a GraphQL request, I think we should have that precise language in the `brief`, for example:

```suggestion
      This span represents an incoming GraphQL request on a server implementation.
```